### PR TITLE
[enterprise-4.11] TELCODOCS-428 RN

### PR DESCRIPTION
**Pull request by kdrake20**
Fixes: https://issues.redhat.com/browse/TELCODOCS-428

For: Version 4.11 

Signed-off-by: Katie Tothill ktothill@redhat.com

### CODE REVIEW

**Comment by ktothill** [comment]
**Who:** Steve Gordon,SME
**Contact Method:** Slack, Direct Message
**Date and Time:** June 22, 2022  16:06
**Summary:** Requested @xsgordon's approval of this release note.

**Next Steps:** Review Steve’s feedback and implement if needed.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -74,6 +74,8 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
 [id="ocp-4-11-installation-and-upgrade"]
 === Installation and upgrade
 
+This release updates the minimum system requirements for installing OpenShift on a single node. Users should configure a minimum of 16 GB of RAM. Specific workload requirements may require additional RAM. The complete list of supported platforms has been updated to include bare metal, vSphere, Red Hat OpenStack, and Red Hat Virtualization platforms. In all cases, the user must specify the `platform.none: {}` parameter in the `install-config.yaml` configuration file.
```
Comment (romfreiman) [inline release_notes/ocp-4-11-release-notes.adoc:77]: is it clear that that 2nd sentence refers to SNO?
Also, I would add something like:

The user must specify the `platform.none: {}` parameter in the `install-config.yaml` configuration file in a case that openshift-installer binary is being used to install single node OpenShift.

**Comment by romfreiman** [comment]
lgtm

File: release_notes/ocp-4-11-release-notes.adoc (hunk 1 above)
Comment (ktothill) [reply release_notes/ocp-4-11-release-notes.adoc]: Updated as suggested.

**Comment by ktothill** [comment]
@xsgordon @romfreiman could you please confirm if the content is now ready for release? Thank you!

**Comment by ktothill** [comment]
**Who:** 
Steve Gordon, SME
Rom Freiman, SME
**Date and Time:** July 5, 2022  14:35
**Summary:** Requested [@xsgordon](https://github.com/xsgordon) and [Rom Freiman](https://issues.redhat.com/secure/ViewProfile.jspa?name=romfreiman)'s approval of release note [PR 46992](https://github.com/openshift/openshift-docs/pull/46992).

**Next Steps:** Review feedback and implement if needed.

**Comment by xsgordon** [comment]
LGTM

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -74,6 +74,8 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
 [id="ocp-4-11-installation-and-upgrade"]
 === Installation and upgrade
```
Comment (bmcelvee) [inline release_notes/ocp-4-11-release-notes.adoc:76]: Please include a title for this release note.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 3]
```diff
@@ -74,6 +74,8 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
...
 
+This release updates the minimum system requirements for installing OpenShift on a single node. When installing OpenShift on a single node, users should configure a minimum of 16 GB of RAM. Specific workload requirements may require additional RAM. The complete list of supported platforms has been updated to include bare metal, vSphere, Red Hat OpenStack, and Red Hat Virtualization platforms. In all cases, the user must specify the `platform.none: {}` parameter in the `install-config.yaml` configuration file in a case that openshift-installer binary is being used to install single node OpenShift.
```
Comment (bmcelvee) [inline release_notes/ocp-4-11-release-notes.adoc:77]: For each reference to the product OpenShift:
s/OpenShift/{product-title}

s/users should/you should 

s/may require/can require

s/the user must/you must

Consider using the replaceables for other Red Hat products too: 
E.g., s/Red Hat OpenStack/{rh-openstack-first}

Add code formatting (backticks): 
s/openshift-installer/`openshift-installer`

**Comment by bmcelvee** [comment]
Hi @ktothill just have a few requested edits. Also please add a link to the build preview for future PRs. Thanks! :)

File: release_notes/ocp-4-11-release-notes.adoc (hunk 2 above)
Comment (ktothill) [reply release_notes/ocp-4-11-release-notes.adoc:76]: Updated as suggested.

File: release_notes/ocp-4-11-release-notes.adoc (hunk 3 above)
Comment (ktothill) [reply release_notes/ocp-4-11-release-notes.adoc:77]: Updated as suggested.

**Comment by ktothill** [comment]
**Contact Method:** Slack, #forum-docs-review
**Date and Time:** July 18, 2022  14:50
**Summary:** Fixed merge conflict and [requested merge](https://coreos.slack.com/archives/C025XCRRH28/p1658170230281029?thread_ts=1657309852.472379&cid=C025XCRRH28).

**Next Steps:** Once merged, Jira can be closed.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 4]
```diff
@@ -68,6 +68,8 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-11-installation-and-upgrade"]
 === Installation and upgrade
 
+==== New minimum system requirements for installing {product-title} on a single node
+This release updates the minimum system requirements for installing {product-title} on a single node. When installing {product-title} on a single node, you should configure a minimum of 16 GB of RAM. Specific workload requirements can require additional RAM. The complete list of supported platforms has been updated to include bare metal, vSphere, {rh-openstack-first}, and Red Hat Virtualization platforms. In all cases, you must specify the `platform.none: {}` parameter in the `install-config.yaml` configuration file in a case that `openshift-installer` binary is being used to install single node {product-title}.
```
Comment (stevsmit) [inline release_notes/ocp-4-11-release-notes.adoc:72]: I think there is an attribute for SNO now: 

:sno: single-node OpenShift

:sno-caps: Single-node OpenShift

You might use it in some cases here (like the last mentioning of single node {product-title}.

Comment (stevsmit) [inline release_notes/ocp-4-11-release-notes.adoc:72]: that the `openshift-installer` binary is ?

Comment (ktothill) [reply release_notes/ocp-4-11-release-notes.adoc:72]: Updated as suggested.

Comment (ktothill) [reply release_notes/ocp-4-11-release-notes.adoc:72]: Updated as suggested.

**Comment by ktothill** [comment]
Merged in [PR 48041](https://github.com/openshift/openshift-docs/pull/48041)

**Pull request closed without merging**